src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -566,6 +570,10 @@ export default Vue.extend({
             })
           })
         })
+      }).catch((error) => {
+        this.showToast({
+          message: error.message
+        })
```

Comment:
@MarmadileManteater This just shows and empty toast, I don't think that `error` object has a `message` property.

Reply:
You are correct. This was a mistake on my part.